### ISSUE TITLE
[Bug]: BarChartGrouped doesn't work with scaleType: 'time'

### ISSUE THREAD

**Issue by cloudlena** [carbon-design-system/carbon-charts#1373]
### What happened?

`BarChartGrouped` cannot be used in combination with `scaleType: 'time'`. Using the configuration described below, the axes are displayed correctly but there is no data shown. However, if I change `scaleType` to `'labels'`, it displays correctly.

I'd like to use the grouped bar chart to show different values for the same time stamp.

### Version

`@carbon/charts-svelte@1.0.2`

### Data & options used

```js
        const data = [
		{
			group: 'Arc',
			date: '2018-12-31',
			value: 22
		},
		{
			group: 'Regular Shutdown',
			date: '2018-12-31',
			value: 1
		},
		{
			group: 'Arc',
			date: '2019-01-06',
			value: 30
		},
		{
			group: 'Regular Shutdown',
			date: '2019-01-06',
			value: 49
		},
		{
			group: 'Irregular Shutdown',
			date: '2019-01-06',
			value: 5
		}
	];

	const options = {
		title: 'Daily Events',
		axes: {
			left: {
				mapsTo: 'value'
			},
			bottom: {
				mapsTo: 'date',
				scaleType: 'time'
			}
		}
	};
```

**Comment by vgb1993** [comment]
Same issue here, seems none of the charts support this Have you found a workaround?

**Comment by theiliad** [comment]
I don't see how the grouped bar chart would work smoothly with the time-series scale. There are lots of details that I think we might not be taking into account here 🙂 

@vgb1993 @cloudlena could you pls share some screenshots of the types of results you'd expect to see?

**Comment by cloudlena** [comment]
@theiliad, thanks for looking into this!
I cannot share a screenshot as I can't produce such a chart. However, what I'd expect is a chart that looks just like https://charts.carbondesignsystem.com/svelte/?path=/story/simple-charts-bar-simple--vertical-simple-bar-time-series-dense-data but that groups multiple bars together if they share the same timestamp.

Does that help?

**Comment by theiliad** [comment]
> @theiliad, thanks for looking into this!
> 
> I cannot share a screenshot as I can't produce such a chart. However, what I'd expect is a chart that looks just like https://charts.carbondesignsystem.com/svelte/?path=/story/simple-charts-bar-simple--vertical-simple-bar-time-series-dense-data but that groups multiple bars together if they share the same timestamp.
> 
> 
> 
> Does that help?

Again, i have a feeling that you don't actually need need time-series support here. The idea of the groups IMO goes against a continuous axis/scale.

For instance if you'd like to show values (Q1 2020, Q2 2020 and so on), you can easily do that with the current discrete one and providing manual values.

**Comment by cloudlena** [comment]
Yes, that's what I'm doing at the moment. I'd like to show multiple values per day as a grouped bar chart. However, that, for example, doesn't allow me to use a zoom bar, right?

**Comment by cloudlena** [comment]
@theiliad, this video shows what I would like to achieve. The chart there is implemented in [PyQt](https://riverbankcomputing.com/software/pyqtchart/download). It shows a grouped bar chart (grouped by day) which is zoomable by selecting parts of the chart by mouse. It would be wonderful to have that some possibility with Carbon charts. 

Additionally, it would be nice to have the x-axis show the nice `time` scale and to be able to show the zoom mini map on top of the chart.

https://user-images.githubusercontent.com/6780471/174300452-d607a2a8-0067-40d0-831b-3e40fad2f0b6.mp4

**Comment by theiliad** [comment]
Closed by #1398

**Comment by cloudlena** [comment]
Awesome! Thanks so much, @theiliad :tada:
